modules/apps/headless/headless-portal-instances/headless-portal-instances-impl/rest-openapi.yaml

```diff
@@ -0,0 +1,204 @@
+components:
+    schemas:
+        PortalInstance:
+            # @review
+            properties:
+                active:
+                    readOnly: true
+                    type: boolean
+                companyId:
+                    format: int64
+                    type: integer
+                domain:
+                    description:
+                        domain used for email authentication.
+                    type: string
+                virtualhost:
+                    type: string
+                webId:
+                    type: string
+            type: object
+info:
+    description:
+        "A Java client JAR is available for use with the group ID 'com.liferay', artifact ID
+        'com.liferay.headless.admin.taxonomy.client', and version '2.0.1'."
+    license:
+        name: Apache 2.0
+        url: http://www.apache.org/licenses/LICENSE-2.0.html
+    title: Liferay Portal Instances Headless API
+    version: v1.0
+openapi: 3.0.1
+paths:
+    /portal-instance/{webId}/activate:
```

Comment:
oh, ok, it's webId. This should be portalInstanceId and ideally in the model use id or portalInstanceId

Reply:
A portal instance (company for friends) has two unique ids: companyId and webId. CompanyId is a numeric value used as primary key in the database, webId is a literal which identifies this portal instance.

I would like to use the webId for this case since it's the value which third party platforms (DXP Cloud) will store, isn't that ok?

I can rename the companyId by id if you consider that necessary but keeping it as companyId was good to let other platforms know the concept of that field.